src/main/java/org/eclipsefoundation/adopters/service/impl/DefaultAdopterService.java

```diff
@@ -58,123 +56,110 @@
 public class DefaultAdopterService implements AdopterService {
 	private static final Logger LOGGER = LoggerFactory.getLogger(DefaultAdopterService.class);
 
-	@ConfigProperty(name = "eclipse.adopters.dirpath", defaultValue = "/tmp/quarkus")
-	String adoptersLocation;
-	@ConfigProperty(name = "eclipse.adopters.filename", defaultValue = "adopters.json")
-	String adoptersName;
+	@ConfigProperty(name = "eclipse.adopters.dirpath", defaultValue = "/tmp/quarkus/adopters.json")
```

Comment:
`dirpath`suffix is now misleading. change to something more explicit (`jsonfile`, `path.json`, ...)